src/Integration.Vsix/ErrorList/SonarErrorListDataSource.cs

```diff
@@ -249,6 +251,21 @@ public bool Contains(IAnalysisIssueVisualization issueVisualization)
 
         #endregion IIssueLocationStore implementation
 
+        #region IClientIssueStore implementation
+
+        public IEnumerable<IAnalysisIssueVisualization> Get()
+        {
```

Comment:
I forgot if I was supposed to add a "var currentFactories = this.factories" before the loop or not